docs/source/gbd2019_models/causes/ihd/index.rst

```diff
@@ -116,331 +112,224 @@ Assumptions and Limitations
-
-.. list-table:: Transition Data
-   :widths: 10 10 10 20 30
-   :header-rows: 1
-   
-   * - Transition
-     - Source 
-     - Sink 
-     - Value
-     - Notes
-   * - r
-     - I
-     - R
-     - remission_rate_cid :math:`= \frac{\text{365 person-days}}{\text{average case duration in days} \times \text{1 year}}`
-     - 
-	 
+     - A
+     - incidence_c493
+     -
+   * - 2
+     - A
+     - P
+     - 28 days
+     - duration-based transition from acute state then progress into post state
+   * - 3
+     - P
+     - A
+     - incidence_493
+     -
+   * - 4
+     - S2
+     - A2
+     - incidence_493
+     -
 
 Data Sources
 """"""""""""
 
-This table contains the data sources for all the measures. The table structure and common measures are as below:
-
-.. list-table:: Data Sources
-   :widths: 20 25 25 25
+.. list-table:: Data Sources and Definitions
+   :widths: 10 10 20 20
    :header-rows: 1
-   
-   * - Measure
-     - Sources
-     - Description
-     - Notes
-   * - prevalence_cid
-     - 
-     - 
-     - 
-   * - birth_prevalence_cid
-     - 
-     - 
-     -
-   * - deaths_cid
-     - 
-     - 
-     - 
-   * - population
-     - 
-     - 
-     - 
-   * - sequelae_cid
-     - 
-     - 
-     - 
-   * - incidence_rate_cid
-     - 
-     - 
-     - 
-   * - remission_rate_m1594
-     - 
-     - 
-     - 
-   * - disability_weight_s{`sid`}
-     - 
-     - 
-     - 
-   * - prevalence_s{`sid`}
-     - 
-     - 
-     - 
 
-An example, that contains common sources for the measures,
-
-.. list-table:: Data Sources
-   :widths: 20 25 25 25
-   :header-rows: 1
-   
-   * - Measure
-     - Sources
+   * - Variable
+     - Source
      - Description
      - Notes
-   * - prevalence_cid
+   * - prevalence_c493
      - como
-     - Prevalence of cause
-     - 
-   * - birth_prevalence_cid
-     - como
-     - Birth prevalence of cause
+     - prevalence of ischemic heart disease
      -
-   * - deaths_cid
+   * - deaths_c493
      - codcorrect
-     - Deaths from cause
-     - 
+     - Count of deaths due to ischemic heart disease
+     -
    * - population
      - demography
-     - Mid-year population for given age/sex/year/location
-     - 
-   * - sequelae_cid
-     - gbd_mapping
-     - List of sequelae
-     - 
-   * - incidence_rate_cid/mid
-     - como/dismod
-     - Incidence rate for cause
-     - 
-   * - remission_rate_cid/mid
-     - como/dismod
-     - Remission rate for cause
-     - 
-   * - disability_weight_s{`sid`}
+     - Mid-year population for given sex/age/year/location
+     -
+   * - prevalence_s{sid}
+     - como
+     - Prevalence of sequela with id {id}
+     -
+   * - disability_weight_s{sid}
      - YLD appendix
-     - Disability weight of sequela with id `sid`
-     - 
-   * - prevalence_s{`sid`}
+     - Disability weight of sequela with id {id}
+     -
+   * - incidence_493
      - como
-     - Prevalence of sequela with id `sid`
-     - 
+     - Incidence of ischemic heart disease
```

Comment:
IHD is probably prevalent enough in older ages to warrant including the correction to go from "population rate" to "susceptible rate", i.e. i = incidence_493 / (1 - prevalence_493)

Reply:
Edit pull request to reflect update to incidence calculation.